src/document/reducer.ts

```diff
@@ -152,25 +181,58 @@ function _baseReducer<T, A extends Action, L>(
  */
 export function baseReducer<T, A extends Action, L>(
     document: Document<T, A, L>,
-    action: A | BaseAction,
+    dispatchedAction: A | BaseAction,
```

Comment:
Why the name change? This might be a bit confusing as there is also a `dispatch` parameter

Reply:
fixed in [b39596d](https://github.com/powerhouse-inc/document-model/pull/29/commits/b39596da1c0fad4c742170a6c43134c4799d830f)